Sources/Profile.php

```diff
@@ -886,6 +886,17 @@ function loadCustomFields($memID, $area = 'summary')
 	$context['custom_fields_required'] = false;
 	while ($row = $smcFunc['db_fetch_assoc']($request))
 	{
+		$value = $user_profile[$memID]['options'][$row['col_name']];
+		$currentKey = 0;
+		if (!empty($row['field_options'])) {
```

Comment:
Nits: 
1. Please move opening braces to a new line.
2. Please remove braces entirely for the `if (empty($currentKey))` block.

Reply:
Arrghh, feels so wrong, but ok :)

Reply:
In SMF 3, I'd like to change the coding style for everything. But that's not where we are now.